### PR TITLE
added repo finder

### PR DESCRIPTION
**Pull request by clsource**
Added a repo finder using the github api

imagen

### CODE REVIEW

**Comment by vaibhavmule** [comment]
This is just wow, can't wait to pull in.

**Comment by clsource** [comment]
cool 👍 
maybe I should replace the node modules with cdn versions of the scripts 😄

**Comment by vaibhavmule** [comment]
What is a use of Filter button?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.DS_Store
```
Comment (vaibhavmule) [inline .gitignore:1]: How about moving this file to root?

File: finder/index.html [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+        <section id="search" style="display:none">
+            <form class="form">
+                <div class="columns is-centered" id="search-div">
+                    <div class="column is-half has-text-centered">
+                        <input  id="searchbar" class="input is-rounded" autofocus="true" placeholder="What you are looking for?" type="text" value="">
+                    </div>
+                </div>
+                <div class="columns is-centered" id="searchbar">
+                    <div class="column is-half has-text-centered">
+                        <button class="button is-info is-large" id="searchbutton">Filter</button>
```
Comment (vaibhavmule) [inline finder/index.html]: How about renaming to search if it is doing work of search rather than the filter.

File: finder/index.html [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+        </section>
+        <div class="margin-bottom-50px"></div>
+    </footer>
+
+    <!-- Scripts -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.22.2/moment.min.js"></script>
+
+    <!-- Fetch Repos -->
+    <script type="text/javascript">
```
Comment (vaibhavmule) [inline finder/index.html]: I would suggest moving js code to file.

File: finder/index.html [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+
+        <section id="results" style="display:none">
+            <table class="table is-striped is-hoverable" id="table-projects">
+                <thead>
+                    <tr>
+                    <th>Name</th>
+                    <th>Description</th>
+                    <th>License</th>
+                    <th>Stars</th>
+                    <th>Url</th>
```
Comment (vaibhavmule) [inline finder/index.html]: capitalize each letter to URL.

File: finder/index.html [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+        })
+    </script>
+
+    <!-- Filter Table -->
+    <script type="text/javascript">
+
+        function reset() {
+
+            // Found here https://www.w3schools.com/howto/howto_js_filter_table.asp
+            let tr = document.getElementsByClassName("repo")
```
Comment (vaibhavmule) [inline finder/index.html]: This is good, how about getting rid of jquery, may be after this is merged?

File: finder/index.html [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+
+                    data.items.forEach(element => {
+
+                        let license = 'Unknown'
+                        if(element && element.license && element.license.name)
+                        {
+                            license = element.license.name
+                        }
+
+                        let updated = moment(element.updated_at || new Date()).format('YYYY-MM-DD')
```
Comment (vaibhavmule) [inline finder/index.html]: remove moment, 

get inspiration from https://stackoverflow.com/questions/23593052/format-javascript-date-to-yyyy-mm-dd

File: finder/package.json [hunk 7]
```diff
@@ -0,0 +1,7 @@
+{
+  "dependencies": {
+    "bulma": "^0.7.1",
+    "jquery": "^3.3.1",
```
Comment (vaibhavmule) [inline finder/package.json]: We should get rid of jquery but after this PR is merged.

**Review by vaibhavmule — CHANGES_REQUESTED** [review]

File: finder/index.html (hunk 2 above)
Comment (clsource) [reply finder/index.html]: done

File: finder/index.html (hunk 3 above)
Comment (clsource) [reply finder/index.html]: done

File: finder/index.html (hunk 4 above)
Comment (clsource) [reply finder/index.html]: done

File: finder/index.html (hunk 6 above)
Comment (clsource) [reply finder/index.html]: done

File: finder/package.json (hunk 7 above)
Comment (clsource) [reply finder/package.json]: need to find a way of doing get requests without much hazzle 👍

Comment (clsource) [reply finder/package.json]: maybe using fetch https://developers.google.com/web/updates/2015/03/introduction-to-fetch

**Comment by clsource** [comment]
Replaced most jquery with vanilla js
and for detecting the input event and document ready
used the zepto js lib https://zeptojs.com/

File: finder/index.html [hunk 8]
```diff
@@ -82,7 +82,7 @@
     </footer>
 
     <!-- Scripts -->
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/zepto/1.2.0/zepto.min.js"></script>
```
Comment (vaibhavmule) [inline finder/index.html:85]: Look good project to me.

**Comment by clsource** [comment]
cool 😎

**Comment by clsource** [comment]
close #2

**Comment by vaibhavmule** [comment]
Let's close #5 this and it will be available at, awesome.masoniteproject.com/finder or /find

**Pull request merged**